src/dimo.test.ts

```diff
@@ -1,76 +1,75 @@
-import { DIMO } from './dimo';
+import { DIMO } from "./dimo";
 
-const PROD = 'Production';
-const DEV = 'Dev';
+const PROD = "Production";
+const DEV = "Dev";
 
-const dimo = new DIMO(PROD);
-const devDimo = new DIMO(DEV);
+// const dimo = new DIMO(PROD);
```

Comment:
It doesn't look like these tests are going to run

Reply:
yeah this is all from what James had done previously.. commenting out for now!